patches/3.6.0.0/Dockerfile

```diff
@@ -0,0 +1,28 @@
+# Fixes to the default 3.6.1.2 reduced image.
```

Comment:
Everything in this file should be moved into the appropriate Ansible playbook or it will cause collisions.
Also, this appears to be the wrong patch directory.  Please revert this file to its historical values.